Packs/SekoiaXDR/Integrations/SekoiaXDR/README.md

```diff
@@ -0,0 +1,893 @@
+
+***
+Command to remove an attribute from an asset in Sekoia XDR. Note: use !sekoia-xdr-get-asset to find the attribute_uuid to delete.
+
+#### Base Command
+
+`sekoia-xdr-remove-attribute-asset`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| asset_uuid | UUID of the asset, the UUID should appear with "sekoia-xdr-list-assets" if that alert have assets related, example: "d4cc3b05-a78d-4f29-b27c-c637d86fa03a". | Required | 
+| attribute_uuid | UUID of the attribute to delete. Note: use !sekoia-xdr-get-asset to find the attribute_uuid to delete. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+### sekoia-xdr-remove-key-asset
+
+***
+Command to remove a key from an asset in Sekoia XDR. Note: use !sekoia-xdr-get-asset to find the key_uuid to delete.
+
+#### Base Command
+
+`sekoia-xdr-remove-key-asset`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| asset_uuid | UUID of the asset, the UUID should appear with "sekoia-xdr-list-assets" if that alert have assets related, example: "d4cc3b05-a78d-4f29-b27c-c637d86fa03a". | Required | 
+| key_uuid | UUID of the key to remove. Note: use !sekoia-xdr-get-asset to find the key_uuid to delete. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+### sekoia-xdr-http-request
+
+***
+Command that performs a HTTP request to Sekoia using the integration authentication configured.
+
+#### Base Command
+
+`sekoia-xdr-http-request`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| method | Method to use with the http request (GET,POST,etc). Default is GET. | Required | 
+| url_sufix | The URL sufix after https://api.sekoia.io/v1, i.e. /sic/alerts/ or /asset-management/assets/. | Required | 
+| parameters | Query parameters, i.e. limit -&gt; 10 , match['status_name'] -&gt; Ongoing. | Optional | 
+
+#### Context Output
+
+There is no context output for this command.
+## Incident Mirroring
+
+You can enable incident mirroring between Cortex XSOAR incidents and Sekoia XDR corresponding events (available from Cortex XSOAR version 6.0.0).
+To set up the mirroring:
+1. Enable *Fetching incidents* in your instance configuration.
+2. In the *Mirroring Direction* integration parameter, select in which direction the incidents should be mirrored:
+
+    | **Option** | **Description** |
+    | --- | --- |
+    | None | Turns off incident mirroring. |
+    | Incoming | Any changes in Sekoia XDR events (mirroring incoming fields) will be reflected in Cortex XSOAR incidents. |
+    | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in Sekoia XDR events (outgoing mirrored fields). |
+    | Incoming and Outgoing |  |
+
+3. Optional: Check the *Close Mirrored XSOAR Incident* integration parameter to close the Cortex XSOAR incident when the corresponding event is closed in Sekoia XDR.
```

Comment:
```suggestion
3. Optional: Check the *Close Mirrored XSOAR Incident* integration parameter to close the Cortex XSOAR incident when the corresponding event is closed in Sekoia XDR.
4. Optional: Check the *Reopen Mirrored Cortex XSOAR Incidents* integration parameter to reopen the Cortex XSOAR incident when the matching Sekoia XDR alert is reopened.
```

Reply:
I think no need for this, in case we are deleting the reopening use case